syntax = "proto3";

package pr_transport_router;

message RouterSettings {
    uint32 bus_wait_time = 1;
    double bus_velocity = 2;
};
